game/scripts/npc/abilities/luna_eclipse.txt

```diff
@@ -25,22 +25,22 @@
 		"AbilityCastPoint"				"0.6 0.6 0.6 0.6 0.6"
 		"AbilityCastRange"				"0"
 
-		// Time		
+		// Time
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityCooldown"				"140.0"
 
 		// Cost
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilityManaCost"				"150 200 250 350 750"
-		
+
 		// Special
 		//-------------------------------------------------------------------------------------------------------------
 		"AbilitySpecial"
 		{
 			"01"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"beams"						"5 8 11 17 40"
+				"beams"						"5 8 11 13 15"
```

Comment:
no, since lucent beam also scales, scaling both leads to silly things. this progression is mildly above the average progression of most spells